docs/identity_management.md

```diff
@@ -0,0 +1,95 @@
+# Identity Management
```

Comment:
can you drop a link in the getting started guide to this ?

Reply:
yep added. hopefully that's the correct way to link it